docs/template.go

```diff
@@ -2,14 +2,15 @@ package main
 
 import (
 	"fmt"
-	"github.com/peterebden/go-cli-init"
 	"hash/adler32"
 	"io/ioutil"
 	"os"
 	"path/filepath"
 	"regexp"
 	"strings"
 	"text/template"
+
+	"github.com/peterebden/go-cli-init/v3"
```

Comment:
What does this do that go-flags doesn't do? Maybe we can merge these?

Reply:
It has a wrapper for flag parsing which we could maybe roll in. Mostly it's more about tying up logging with it for a bunch of niceties that we can share between multiple projects.